operators/config/dev/default-storage.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: storage.k8s.io/v1
```

Comment:
Maybe add a comment:
`# Replace GKE default storageClass to set "volumeBindingMode: WaitForFirstConsumer"`